package.json

```diff
@@ -23,11 +31,6 @@
   "engines": {
     "node": ">=0.6"
   },
```

Comment:
@PhilipGough AIUI, this package.json doesn't actually get used at all. It's the ones in the individual components that are important